### PR TITLE
(SIMP-5585) Test assets support Puppet 5

### PR DESCRIPTION
**Pull request by op-ct**
SIMP-5585 #comment Added pup5 test assets to simplib

### CODE REVIEW

**Comment by op-ct** [comment]
⏳ https://gitlab.com/simp/pupmod-simp-simplib/pipelines/35541079

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,479 +1,314 @@
...
-# by non-EOL meta-releases of SIMP and LTS versions of Puppet Enterprise.
+# ==============================================================================
 
-# ----------------------------------------------------------------------
-# Puppet 4.8 for SIMP 6.0 and PE 2016.5
-pup4_8-default:
+pup4.10:
   <<: *acceptance_base
-  <<: *simp_6_0
   <<: *only_with_SIMP_FULL_MATRIX
```
Comment (lnemsick-simp) [inline .gitlab-ci.yml]: At least 1 4.10 test needs to be run outside of the full matrix

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -1,479 +1,314 @@
...
-  <<: *only_with_SIMP_FULL_MATRIX
-  variables:
-    BEAKER_fips: 'yes'
-  script:
-    - 'bundle exec rake beaker:suites[default]'
-
-pup5_5-default-oel:
-  <<: *acceptance_base
-  <<: *pe_2018_1
+  <<: *pup_5_5_7
```
Comment (lnemsick-simp) [inline .gitlab-ci.yml]: At least 1 OEL test needs to be run outside of the full matrix

**Comment by lnemsick-simp** [comment]
:hourglass_flowing_sand: https://gitlab.com/simp/pupmod-simp-simplib/pipelines/35752395

**Comment by lnemsick-simp** [comment]
https://gitlab.com/simp/pupmod-simp-simplib/pipelines/35768324

**Pull request merged**